docs/experiments-sbert.md

```diff
@@ -0,0 +1,35 @@
+# Pyserini: Replicating S-BERT MSMARCO Results
```

Comment:
Do we want to use "SBERT" (without dash) to be consistent with everything else?

Reply:
sure